www/admin/app-admin.less

```diff
@@ -13,8 +13,17 @@
         display: none !important;
     }
 
+    label{
+        margin-top:0.5rem;
+    }
+
     display: flex;
     flex-flow: column;
+
+    .btn-danger{
```

Comment:
This selector is way too broad for the problem being fixed. It affects all `danger` buttons in the admin app, causing alignment issues in some cases such as this one

I think this problem may have been part of the regression but if not it will need a more specific selector for the context you are trying to solve

![image](https://github.com/cryptpad/cryptpad/assets/11028490/797d9499-2e20-4773-9cc9-4810c69ac736)